core/platform/transactions/cloud_transaction_services.py

```diff
@@ -23,13 +23,10 @@
 
 from google.cloud import datastore
 
-from typing import Any, Callable # isort:skip # pylint: disable=unused-import
```

Comment:
This file was filled with usages of unexplained Any. So I decided to keep this file in denylist only (as it is now), removed type annotations with unexplained Any, but kept the remaining annotations (but there weren't in this case).